sdk/docs/iot/resources/iot_hub_flow.puml

```diff
@@ -74,7 +74,7 @@ az_iot_hub_client_sas_get_signature : - token_expiration_epoch_time
 az_iot_hub_client_sas_get_password: - Base64(HMAC-SHA256(signature, SharedAccessKey))
 az_iot_hub_client_sas_get_password: - key_name
 
-az_iot_hub_client_telemetry_get_publish_topic : - az_iot_hub_client_properties

```

Comment:
Can you provide some context, what motivated this change?

Reply:
Context here or in some doc or change-log? At least for here, reason being that come IoT Plug and Play, this will be a commonality: both hub client and pnp client will need some mechanism to create and use properties. It therefore isn't particular just to iot hub client and so it is put here in common.

Reply:
> Context here or in some doc or change-log? 

Sorry, I mean, the context behind this whole PR, i.e. why we are making the change overall. Why are we renaming `az_iot_hub_client_properties`? Was it based on some feedback?

Reply:
Gotcha. Yea for the reasons above. It will be used with two clients and we wouldn't want to have duplicate code